features/facebookImport/src/locales/en/miniStories/activities.json

```diff
@@ -1,6 +1,6 @@
 {
-    "title": "Your activities",
-    "summary": "Since you joined Facebook recorded {{number_activities}} activities of yours with date and time.",
+    "title": "Data from you and about you",
+    "summary": "Since you joined Facebook recorded {{number_activities}} data events from your and about you with date and time.",
```

Comment:
from your what?

Reply:
timeline maybe?

Reply:
I think it's meant to say "from you and about you" (but yeah, typo :D)

Reply:
That's literally what it says :D 

I think you mean "from and about you" right?

Reply:
It says "from your and about you"